ios/tools/prepRelease.sh

```diff
@@ -68,16 +70,17 @@ FRAMEWORK="KeymanEngine.xcframework"
 KEYMAN_SAMPLES="samples"
 
 echo "engine dest: $KMEI_DST"
+ZIP_FLAGS=("-q" "-r" "-X") # quiet, recursive, no-extra
 
 echo "Zipping ${FRAMEWORK} => ${UPLOAD_DIR}/${KMEI_DST}..."
 cd "${KMEI_FRAMEWORK_BASE}"
-zip -qrX "${KMEI_DST}" ${FRAMEWORK}
+zip_files "${KMEI_DST}" "${ZIP_FLAGS[@]}" "${FRAMEWORK}"
```

Comment:
Now that I see how it's used I stumble on the name `zip_files`. Maybe we should rename the function to something like `add_files_to_zip`, `add_zip_files`, `create_zip`,... ?

Reply:
I went with `add_zip_files`. 

Later when we implement extract, we can use `extract_zip_files`

Reply:
We won't need extract -- `unzip` is already on all our dev platforms by default.